lib/mpool/src/mblock_fset.c

```diff
@@ -870,7 +870,7 @@ mblock_fset_clone(
     if (err)
         goto errout;
 
-    err = mblock_file_wlen_set(tgt_mbfp, tgt_mbid, off + len);
+    err = mblock_wlen_set(tgt_mbfp, tgt_mbid, off + len, 0);
```

Comment:
Change last param from `0` to `false`.

Also, is it possible at this call site for `off+len` to exceed `tgt_mbfp->mblocksz`?

If not, then consider changing `mblock_wlen_set` to return `void` and change the param checks to `INVARIANT` macros.

Reply:
Done, thnx!